README.md

```diff
@@ -44,7 +44,7 @@ compatibility of one minor version between client and server:
 | master | 1.14.1  | v11.0.0          | release-1.14        | 1.13.x, 1.14.x, 1.15.x            |
 | 1.2.0  | 1.13.6  | v10.0.0          | release-1.13        | 1.12.x, 1.13.x, 1.14.x            |
 | 1.1.0  | 1.12.1  | v9.0.0           | release-1.12        | 1.11.x, 1.12.x, 1.13.x            |
-| 1.0.0  | 1.7.6   | v4.0.0           | release-1.7         | 1.6.x, 1.7.x, 1.8.x               | 
+| 1.0.0  | 1.7.6   | v4.0.0           | release-1.7         | 1.6.x, 1.7.x, 1.8.x               |
```

Comment:
spurious change?

Reply:
Has been part of 048bba446f443d694ae121def56fa2bf48c71488 (https://github.com/weaveworks/kured/blame/master/README.md#L47).